lib/policy_machine/policy_element.rb

```diff
@@ -249,6 +262,22 @@ def self.all(pm_storage_adapter, options = {})
       
     end
 
+    def to_s
+      unique_identifier
+    end
+
+    def operation
```

Comment:
Should these methods (other than <code>prohibition</code> which is covered) have specs